Exit with an error code when generate command fails. (#969)
realm/realm-dart: CHANGELOG.md

### Fixed
* Added more validations when using `User.apiKeys` to return more meaningful errors when the user cannot perform API key actions - e.g. when the user has been logged in with API key credentials or when the user has been logged out. (Issue [#950](https://github.com/realm/realm-dart/issues/950))

> * Fixed `dart run realm_dart generate` and `flutter pub run realm generate` commands to exit with an error code when generation fails. Execution is now terminated when the command is run in a batch with other commands.

Reviewer: ```suggestion
* Fixed `dart run realm_dart generate` and `flutter pub run realm generate` commands to exit with the correct error code when generation fails.
```
